README.md

```diff
@@ -1,133 +1,148 @@
+
+**Open the cloned repository with Visual Studio Code**
+
+<img src="./images/code_open.PNG" width="75%">
+
+**Visual Studio Code opens**
+
+<img src="./images/vscode_opens.PNG" width="75%">
+
+| Note: the bottom left green button shows the Visual Studio Remote Development extension has been installed
+
+**Click on the button "Reopen in Container"**
+
+<img src="./images/vscode_reopen_container.PNG" width="75%">
+
+While Visual Studio Code reopens your project and load the Azure CAF rover you will see the following icon
+
+<img src="./images/vscode_opening_remote.PNG" width="75%">
+
+| Note: The first time it will take longer as the full docker image has to be downloaded.
+
+When successfuly loaded you will see Visual Studio Code opened with the following look and feel
+
+<img src="./images/vscode_opened.PNG" width="75%">
+
+It is recommeded you leverage the workspace in order to drive more consistancy across different operating systems
+
+<img src="./images/vscode_container_ws.PNG" width="75%">
+
+## Login the rover to Azure
+Open a new terminal from the menu Terminal..New Terminal
+
+Run **Rover login**
+
+<img src="./images/rover_login.PNG" width="75%">
+
+Note: If you have more than one subscription or Azure AD Tenant you can use the command: 
+```bash
+# Login to your Azure subscription --> if you have multiple subscriptions and tenant, set the subscription guid and tenant guid
+rover login [subscription_guid] [tenantname.onmicrosoft.com or tenant_guid]
+# Verify the current Azure subscription
+az account show
+```
+
+**Authenticate with your credential**
+
+<img src="./images/rover_login1.PNG" width="75%">
+
+Note: Copy the code and open the device login to set your username and password
+
+
+<img src="./images/rover_logged.PNG" width="75%">
+
+## Initialize the Level0 launchpad
+
+Initialize the rover by calling the rover command. The initialization of the launchpad will take between 5-10 mins.
+
+```bash
+# Deploy the launchpad level0
+rover
+```
+
+## Deploy the Cloud Adoption Framework foundations landing zone: 
+```bash
+# Display the resources that are going to be deployed
+rover /tf/landingzones/landingzone_caf_foundations plan
+
+# Deploy the resources
+rover /tf/landingzones/landingzone_caf_foundations apply
+
+```
+
+
+### Something you want, something not working?
+Open an issue list to report any issues and missing features.
+
+
+## Troubleshooting
+### Error codes
+Error code returned by the bash (echo $?)
+
+| Code | Description | 
+|--- |--- |
+| 0 | Operation completed successfully 
+|2 | Not connected to Azure subscription. You need to logout / login and set the default subscription 
+|10 | Launchpad is installed but no landingzone and action arguments have been set 
+|11 | Landingzone argument set without an action 
+|12 | Landingzone folder does not exist in the rover 
+
+### Purging Docker cache
+You can purge Docker cache running the following command:
+```bash
+docker system prune -a
```

Comment:
Should we as well add a command to remove the unused volume (if using vs code + docker remote env) but without using prune as it could have side effects on other volumes:

docker volume rm -f $(docker volume ls -f label=com.docker.compose.project=landingzones_devcontainer)

Reply:
Good catch @nyuen

Reply:
@nyuen Added the cleaup in the doc